leftwm/src/bin/leftwm.rs

```diff
@@ -55,12 +58,17 @@ fn main() {
         env::set_var("_JAVA_AWT_WM_NONREPARENTING", "1");
 
         let worker_path = current_exe.with_file_name("leftwm-worker");
+        let lefthk_path = current_exe.with_file_name("lefthk-worker");
```

Comment:
I am thinking if this should possibly be ̀ hotkeydaemon_path` or something. Opinions?

Reply:
Another thought in this direction, maybe this is stupid though: rename `lefthk-worker` to `lefthk-daemon` alltogether?